docs/README.md

```diff
@@ -43,7 +43,7 @@ Akri's documentation is divided into six sections:
 
 1. 📘 [User Guide](./user-guide/getting-started.md): Documentation for Akri users.
 1. 🔎 [Discovery Handlers](./discovery-handlers/onvif.md): Documentation on how to configure Akri using Akri's currently supported Discovery Handlers
-1. 🚀 [Demos](./demos/usb-camera-demo.md): End-to-End demos that demostrate how Akri can discover and use devices. Contain sample brokers and end applications.
+1. 🚀 [Demos](./demos/video-links.md): Overall introduction videos and end-to-end demos that demostrate how Akri can discover and use devices. Contain sample brokers and end applications.
```

Comment:
I'd prefer to leave this as linking to the usb demo and keeping that first under the section as it is developer documentation

Reply:
Reversed this link and description.